python/mdvtools/conversions.py

```
@@ -328,63 +330,167 @@
-    bed_folder="",
-    openchrome="DNase",
-    openchrome_color="#eb9234",
-    marks=["H3K4me1", "H3K4me3", "H3K27ac", "CTCF"],
-    mark_colors=["#349beb", "#3aeb34", "#c4c41f", "#ab321a"],
-    genome="hg38",
+    table,
+    bigwigs,
+    beds,
+    matrix=None,
+    openchrom="DNase",
+    marks=["ATAC", "H3K4me1", "H3K4me3", "H3K27ac", "CTCF"],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)